src/App.tsx

```diff
@@ -1,109 +1,38 @@
 import React from 'react';
-
+import { PostList } from './components/PostList';
+import { PreparedPost } from './types/PreparedPost';
+import { User } from './types/User';
+import { Comment } from './types/Comment';
 import './App.scss';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
-
-export const App: React.FC = () => (
-  <section className="App">
-    <h1 className="App__title">Static list of posts</h1>
-
-    <div className="PostList">
-      <div className="PostInfo">
-        <div className="PostInfo__header">
-          <h3 className="PostInfo__title">qui est esse</h3>
-
-          <p>
-            {' Posted by  '}
-
-            <a className="UserInfo" href="mailto:Sincere@april.biz">
-              Leanne Graham
-            </a>
-          </p>
-        </div>
-
-        <p className="PostInfo__body">
-          est rerum tempore vitae sequi sint nihil reprehenderit dolor beatae
-          ea dolores neque fugiat blanditiis voluptate porro vel nihil
-          molestiae ut reiciendis qui aperiam non debitis possimus qui neque
-          nisi nulla
-        </p>
-
-        <hr />
-
-        <b data-cy="NoCommentsMessage">No comments yet</b>
-      </div>
+function getUser(userId: number):User | null {
```

Comment:
<img width="1103" alt="image" src="https://user-images.githubusercontent.com/91411692/232417571-a1a6d833-3682-4ffe-9edf-4b27bf385258.png">
```suggestion
function getUserById(id: number):User | null {
```